spark/src/test/scala/org/apache/spark/sql/CometTPCHQuerySuite.scala

```diff
@@ -90,10 +90,8 @@ class CometTPCHQuerySuite extends QueryTest with TPCBase with ShimCometTPCHQuery
     conf.set(CometConf.COMET_EXEC_ENABLED.key, "true")
     conf.set(CometConf.COMET_NATIVE_SCAN_ENABLED.key, "true")
     conf.set(CometConf.COMET_EXEC_SHUFFLE_ENABLED.key, "true")
-    conf.set(CometConf.COMET_SHUFFLE_MODE.key, "jvm")
```

Comment:
This is unrelated, but there is no reason for us to disable native shuffle now. I can revert this change and move to a separate PR if needed.